### DISCUSSION TITLE
Raw f Strings in Python

### DISCUSSION THREAD

**Discussion by culpgrant** [astral-sh/ruff#14100] [Q&A]
Why does ruff reformat `fr"my_string and {variable}"` into `rf"my_string and {variable}"`?

I looked through the GitHub and the docs website I was unable to find this as a setting.

**Comment by dhruvmanila** [comment]
(I remember there was an issue / discussion for the same but unable to find it.)

I think that's mainly for consistency, it could be `rf` or `fr` but neither of them changes the meaning of the code. Black also puts it in this specific order ([Black playground](https://black.vercel.app/?version=main&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4ACeAGRdAD2IimZxl1N_WlkB95D1v1roLPOm6P63DiVHTPHwWUh4qeWCiEobCQ9xXFCYhfni9qcBIMIYWtyYLtNWuqyTbKVXcqwnnz8wIC0i3tGZbP3O7SAjiyusF-WFNPB-vhRgg9RdjQAArq27o4QTOzcAAYABnwEAAHByZASxxGf7AgAAAAAEWVo%3D)).

**Reply by culpgrant** [reply]
Yeah it doesn't change the code, I was just wondering. Thank you!
